exp/controllers/awsmachinepool_controller.go

```diff
@@ -353,12 +384,27 @@ func (r *AWSMachinePoolReconciler) reconcileNormal(ctx context.Context, machineP
 		machinePoolScope.Error(err, "failed updating instances", "instances", asg.Instances)
 	}
 
-	return nil
+	if feature.Gates.Enabled(feature.MachinePoolMachines) {
+		return ctrl.Result{
+			// Regularly update `AWSMachine` objects, for example if ASG was scaled or refreshed instances
+			// TODO: Requeueing interval can be removed or prolonged once reconciliation of ASG EC2 instances
```

Comment:
Do we need to create an issue for this?

Reply:
I created https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/5323 and referenced it here